src/borg/helpers/datastruct.py

```diff
@@ -0,0 +1,56 @@
+import threading
+from collections import namedtuple
+
+from .errors import Error
+
+
+ArchiveInfo = namedtuple('ArchiveInfo', 'name id ts')
```

Comment:
it initially was there and i moved to avoid some cyclic import while doing the refactoring.
i can try again if it works now.